Jenkinsfile

```diff
@@ -27,6 +27,15 @@ pipeline {
       }
     }
 
+    stage('Install Correct Node Version') {
+      steps {
+        ansiColor('xterm') {
+          sh 'npm install -g n'
```

Comment:
Why are we using `n` to install "manually" the node version we want? I believe we should have the node version already available in the environment that it's running the jenkinsfile.

e.g: On Jenkins you can configure to run on docker environment `node:8.9.4-alpine`

And `n` hasn't been updated since `Oct 18, 2017` so I believe won't receive maintenance anymore

Reply:
@juliangieseke You had a reason for that right? Was it that we didn't want to change the dockerfile, yet?

Reply:
I'd +1 @weblancaster 's question about `n`. Not sure who active of project it is - especially since original creator TJ has his own company now. We already have a `.nvmrc`, so if we have to use something probably use nvm is better.

Reply:
Also why do we need to use a node version manager at all. Can't we just `npm install -g node@8.9`?

Reply:
Yeah, I agree with y'all it should also already be in the docker container, therefore I'll remove it :)

Reply:
AFAIK npm 3.9 isnt capable of installing node versions - if it is, this would be way better 👍 
but this whole thing is only a dirty hack because we havent changed our Dockerfile yet, we will change this in future to be able to install the correct node version with `npm install` (which depends on #2686)